conanfile.py

```diff
@@ -1,12 +1,12 @@
-from conans import ConanFile
-
+from conans import ConanFile, CMake
```

Comment:
Can that be generated or add a library or form can give the file JSON Extension ?

Reply:
I don't understand this question. What do you mean with JSON extension?